src/components/totals/TotalSupporters/__tests__/TotalSupporters-test.js

```diff
@@ -247,4 +247,28 @@ describe('TotalSupporters', function() {
       expect(icon).not.toBeNull();
     });
   });
+
+  describe('Filtering by page type', function() {
+    var totalSupporters;
+    var element;
+
+    beforeEach(function() {
+      pages.search.mockClear();
+      totalSupporters = <TotalSupporters charityUids={ ["au-24", "au-31"] } type="team" />;
+      element = TestUtils.renderIntoDocument(totalSupporters);
+    });
+
+    it('requests pages only of type "team"', function() {
+      expect(pages.search.mock.calls.length).toEqual(1);
+      expect(pages.search).toBeCalledWith({
+        campaignUid: [],
+        charityUid: ["au-24", "au-31"],
```

Comment:
Is there a way to check that only the charityUids are what you expect? Just so that you don't have to maintain all these default values.